src/main/kotlin/no/nav/familie/ef/sak/blankett/BlankettController.kt

```diff
@@ -26,9 +34,32 @@ class BlankettController(private val tilgangService: TilgangService,
         tilgangService.validerTilgangTilBehandling(behandlingId)
         val blankettPdfRequest = BlankettPdfRequest(lagPersonopplysningerDto(behandlingId),
                                                     hentInngangsvilkårDto(behandlingId))
-        return Ressurs.success(blankettClient.genererBlankett(blankettPdfRequest))
+        val blankett = blankettClient.genererBlankett(blankettPdfRequest)
+        blankettService.oppdaterBlankett(behandlingId, blankett)
+        return Ressurs.success(blankett)
     }
 
+    @PostMapping("/oppgave/{oppgaveId}")
+    fun opprettBlankettBehandling(@PathVariable oppgaveId: Long): Ressurs<UUID> {
```

Comment:
Hva er flytet for denne? :) Hva er det som trigger att man kaller på denne endepunkten?

Reply:
Vi lurer på det samme egentlig - hva er det som trigger at man kaller på dette endepunktet ? Det ble tenkt sammen med @ma10s at vi kan opprette en tom blankett ved opprettelse av _behandlingen_ (og at blanketten oppdateres ved hver generering av blankett (overskrives)). Vet ikke når en behandling opprettes ?

Reply:
Tidligere kunne man ikke gjøre noe med disse sakene. Når man trykket behandle sak fikk man feilmelding opp om at det finnes en sak i infotrygd(?) og kan derfor ikke behandles. Nå får man samme feilmelding, men med knapp som gir muligheten til å opprette blankett. Det er denne knappen som peker til dette endpointet.